network/src/application/netperf/interface.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+//
+use crate::protocols::network::{NetworkEvents, NetworkSender};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct NetPerfPayload {
+    byte: Vec<u8>,
+}
+
+impl NetPerfPayload {
+    pub fn new(mut len: usize) -> Self {
+        let mut v = Vec::with_capacity(len);
+        while len > 0 {
+            v.push(0);
+            len -= 1;
+        }
+        NetPerfPayload { byte: v }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum NetPerfMsg {
+    BlockOfBytes(NetPerfPayload),
+}
+/// The interface from Network to NetPerf layer.
+///
+/// `NetPerfNetworkEvents` is a `Stream` of `PeerManagerNotification` where the
+/// raw `Bytes` rpc messages are deserialized into
+/// `NetPerfMsg` types. `NetPerfNetworkEvents` is a thin wrapper
+/// around an `channel::Receiver<PeerManagerNotification>`.
+pub type NetPerfNetworkEvents = NetworkEvents<NetPerfMsg>;
+
+/// The interface from NetPerf to Networking layer.
```

Comment:
nit: the comments feel a little redundant to me given that `NetworkEvents<T>` and `NetworkSender<T>` are used by all applications, but feel free to leave them if you find them helpful.